### ISSUE TITLE
Nondescript blocker order with face down creatures

### ISSUE THREAD

**Issue by PurpleCrowbar** [magefree/mage#12880]
After declaring the order of blockers when multiple of those blockers are face down creatures, it's impossible to know from the log what the order was. In this case, there was a disguised and morphed creature. Both are just listed as "face down creature" twice in the log. Their ID should be in the blocker order log message. See below
![image](https://github.com/user-attachments/assets/5496a711-ac25-4176-a085-4401c6ee041e)

**Comment by xenohedron** [comment]
Presumably due to this code which overwrites the log name for empty names, without including the id:

https://github.com/magefree/mage/blob/6a19ec48df390a0d8efc4572e7091bb044e0a7c9/Mage/src/main/java/mage/game/permanent/PermanentImpl.java#L1903-L1913

**Comment by Dunkoro** [comment]
Preferably all references to any card in chat should include IDs, just like with targetted cards.

**Comment by JayDi85** [comment]
Yes. Not only logs, but all GUI related data also must use html related "logName", not text related "IdName" -- it's full html with colored name, id and inner ids (popup card hints uses inner html tags to find and show detailed info/ref). Only few exceptions like exile windows naming (it's not support html yet).
